opds.py

```diff
@@ -488,16 +488,17 @@ class AcquisitionFeed(OPDSFeed):
 
     @classmethod
     def groups(cls, _db, title, url, lane, annotator, 
-               force_refresh=False,
+               force_refresh=False, cache_type=None,
```

Comment:
don't understand why cache_type is now an argument to `groups`. I get `page` but not `groups`. Is it just to keep the signatures the same or do you anticipating making both page-type and group-type recommendation feeds? (If you do plan on making both, then 'recommendations' isn't a new type of CachedFeed, it's a separate piece of information that needs to be stored separately from whether the feed is `page` or `groups`.